### PR TITLE
Fix/batch of fixes and implementing enhancements

### LINKED ISSUE #10: Missing the Hover effect
The effect is missing on the Get Bonus Button.

### LINKED ISSUE #11: Missing CTA Content
The title of the button is not loading correctly

### LINKED ISSUE #12: Review link not working
The link review is not working.

### LINKED ISSUE #13: Incorrect Desktop font size
Screenshot 2021-08-15 at 16 55 50

### LINKED ISSUE #14: Add CTA Link
Add CTA-LINK at these

Screenshot 2021-08-15 at 16 58 28
Screenshot 2021-08-15 at 16 58 06
 areas.

### LINKED ISSUE #15: Centered items when only 2 or 1
Here if there are less than 3 should remain centered.

Screenshot 2021-08-16 at 12 30 59

### PR DESCRIPTION
**Pull request by migbash**
+ closes #15 Centered items when only 2 or 1 **enhancement**
+ closes #14 Add CTA Link **enhancement**
+ closes #13 Incorrect Desktop font size bug **fix** - no cost
+ closes #12 Review link not working bug **fix** - no cost
+ closes #11 Missing CTA Content bug **fix** - no cost
+ closes #10 Missing the Hover effect **enhancement**

### CODE REVIEW

File: src/components/_MobileBetarenaWidget.svelte [hunk 1]
```diff
@@ -246,11 +246,13 @@ MOBILE FIRST -->
...
             matchbetting-logo -->
-            <img 
-                src={data.logo}
-                alt=""
-                id='booker-logo'
-            />
+            <a href={ data.cta_link }>
+                <img 
+                    src={data.logo}
+                    alt=""
```
Comment (jonsnowpt) [inline src/components/_MobileBetarenaWidget.svelte]: alt = Betting Site Logo

File: src/components/_TabletPlusBetarenaWidget.svelte [hunk 2]
```diff
@@ -344,11 +345,13 @@ TABLED & DESKTOP FIRST -->
...
             matchbetting-logo -->
-            <img 
-                src={data.logo}
-                alt=""
-                id='booker-logo'
-            />
+            <a href={ data.cta_link }>
+                <img 
+                    src={data.logo}
+                    alt=""
```
Comment (jonsnowpt) [inline src/components/_TabletPlusBetarenaWidget.svelte]: alt = Betting Site Logo

**Review by jonsnowpt — CHANGES_REQUESTED** [review]
We need to add the ALT attributes, from now on, alt must always have an attribute.

File: src/components/_MobileBetarenaWidget.svelte (hunk 1 above)
Comment (migbash) [reply src/components/_MobileBetarenaWidget.svelte]: Applied

File: src/components/_TabletPlusBetarenaWidget.svelte (hunk 2 above)
Comment (migbash) [reply src/components/_TabletPlusBetarenaWidget.svelte]: Applied

**Pull request merged**